src/main/java/frc/robot/lib/states/Arm.java

```diff
@@ -37,4 +74,62 @@ public void moveArmPosition(double firstStage, double secondStage, double wrist)
         RobotMap.armSecondStage.set(TalonFXControlMode.Position, secondStage);
         RobotMap.wrist.set(TalonFXControlMode.Position, wrist);
     }
+
+    /* getClosedLoopError(): Gets the closed-loop error. The units depend on which control mode is in use. 
```

Comment:
This comment is not describing the functionality of the method. Comments in front of methods need to only pertain to describing how that mehtod functions.